### ISSUE TITLE
Not an issue, just a question

### ISSUE THREAD

**Issue by herber4** [gtonkinhill/panaroo#87]
Hello, 

Just a question, 

How would one Identify the single copy orthologs with the Panaroo output?

**Comment by gtonkinhill** [comment]
Hi,

This should probably be a little easier than it is. This information is currently stored in the `final_graph.gml` file and can be viewed in [cytoscape](https://cytoscape.org/) or the python [networkx](https://networkx.org/documentation/networkx-2.5/]) library can be used to traverse the graph.

Single copy orthologs will have a '0' in the 'paralog' field of the nodes in this graph.

**Comment by herber4** [comment]
Awesome, thanks! I was able to extract the list of single copy core genes and am planning on using them downstream in an IQ-Tree analysis. Though I extracted the .aln.fas files that had '0' in the 'paralog' field, some of the alignment files have duplicated seq from the same sample...like such: 

>JVD0051;10_30_11
------------------------------------------------------------
------------------------------------------------------------
------------------------------------------------------------
------------------------------------------------------------
------------------------------------------GTGGAAGAATCCGGTTTC
ATCTTCATCGGCCCCAAGGCCGACACCATTCGCCTGATGGGCGACAAGGTCGAGGCGATC
CGCGCGATGAAGGCCGCCGGCGTGCCGTGCGTGCCCGGCTCGGGTGGTCCGTTGGGCGAT
GACATCGTCGCCAACACCAAGGTAGCGCGCGAAATCGGCTACCCAGTGATCATCAAGGCC
GCCGGCGGCGGCGGCGGGCGCGGTATGCGCGTGGTGCATTCGGAAGCCGCGCTCAAGGCG
GCCATCGAGACCACCAAGTCCGAAGCCAAGGCCGCTTTCAGCAATGATCAGGTCTACATG
GAGAAGTTCCTGGAAAATCCGCGTCACGTGGAAATCCAGGTGCTCGCCGACGGCCAGGGC
AATGCGATCCATCTGGGCGAACGCGATTGCTCGATGCAGCGTCGTCACCAGAAAGTAGTC
GAAGAAGCCCCCGCACCGGGCATTACCGAGGAACTACGCAACGAGATCGGCAAGGTCTGC
GTGGATGCCTGCATCCGCATCGGCTATCGCGGCGCGGGCACCTTCGAATTCCTGTTCGAA
GAGGGGCGTTTCTACTTCATCGAAATGAACACCCGTATCCAGGTGGAGCATCCGGTGACC
GAACGCATCACCGGCATCGACCTGGTGTGCGAGCAGCTGCGCATCGCCGCCGGGCAAAAG
CTGGCCCTCAAGCAGAGCGACATCGTGCTGCGCGGGCATGCGATCGAGTGCCGCATCAAC
GCCGAGGACCCGGAAACCTTCATGCCCAATCCGGGCCTGATCACCGCATTCCATCCGCCC
GGTGGTCCAGGTGTGCGTGTGGATACGCATATCTACGCCGGCTACAAGGTGCCGCCGAAT
TACGACTCCATGATCGGCAAGCTGATCGTGCACGGCCCGGATCGCGAAACCGCGATCGCG
CGCATGCGCGTGGCGCTGAGCGAAATGGTGGTGGACGGCATCAAGACCAACATCCCGCTA
CAGCAGCGCATCATGCGCGACAAGGGCTTCCAGGCCGGTAGGCAGAACATCCACTATCTG
GAAAAGCGCCTGGCCGAGCGCAAGAACAAGTCGATCGCGCTGGTCTGA
>JVD0051;10_31_0
---------GTGATCATCAAGGCCGCCGG-------------------------CGGCCT
TGATGATCACTGGGTAGCCGATTTCGCGCGCTACC--------------TTGGTGTTGGC
GACGATGTCATCGCCCA--------------ACGGACCACCCGAGCCG---GGCACGCAC
GGCACGCCGGCGGCCTTCATCGCGCGGATCGCCTCGACCTTG------------------
----TCGCCCATCAGGCGAATGGTGTCGGCCTTGGGGCCGATGAAGATGAAACCGGATTC
TTCCACCCGCTCGGCAAAGTCGGCGTTTTCCGACA-------------------------
-----GAAAGCCGTAGCCGGGATG------------------------------------
--------------------------------GATGGCCTGCGCATCGGTCAC-------
------------------------------------------------------------
--------------------------CTCGGCCGCGGCGATCAACGCTGGGATATTGAGG
TAACTGTCGCTGGACGCG----------------------------------GCCGGGCC
GATACACACCG------------------------------------------ACTCGTC
GGCCATGGCCACGTGCTTGAG---------------------------------------
------------------------------------------------------------
------------------------------------------------------------
------------------------------------------------------------
------------------------------------GTTGCGGTCGACCGTGGAATGCAC
CGCGACCGTGCGG-----------------------------------------------
-------ATGCCGAGCGTGTGGC-------------------------------------
------------------------------------------------------------
------------------------------------------------------------
-ACGCGCGAAGGATGCGCA--GCGCGATTTCACCTCGGTTGGCGATGACGATTTTATCTA
G-----------------------------------------------

This is jamming up the IQ-Tree pipeline. what parameters would I need to adjust to get rid of these faulty duplicates in the single copy orthologous genes? 

Best, 

Austin
